indico/modules/events/registration/models/registrations.py

```diff
@@ -126,6 +126,12 @@ class Registration(db.Model):
             lazy=True
         )
     )
+    #: If the registration has been deleted
+    is_deleted = db.Column(
```

Comment:
You might want to add it to the repr, too:

    return format_repr(self, 'id', 'registration_form_id', 'email', 'state', user_id=None, is_deleted=False, _text=full_name)